astropy/coordinates/representation.py

```diff
@@ -2029,10 +2068,22 @@ def norm(self):
         """
         return np.abs(self.distance)
 
-    def __neg__(self):
-        self._raise_if_has_differentials('negation')
-        return self.__class__(self.lon + 180. * u.deg, -self.lat, self.distance,
-                              copy=False)
+    def _scale_operation(self, op, *args):
+        # TODO: expand special-casing to UnitSpherical and RadialDifferential.
```

Comment:
This will reduce back to `Radial` or `UnitSpherical`, and, following your suggestion, no longer goes through `Cartesian`, so there is not much of an effect any more.